cmd/server/handlers.go

```diff
@@ -73,11 +77,11 @@ func triggerAction(rule *rule.Rule, event *evt.Event) {
 
 // pingHandler is a simple handler to test if daemon is UP.
 func pingHandler(w http.ResponseWriter, r *http.Request) {
-	w.Write([]byte("pong\n"))
+	_, _ = w.Write([]byte("pong\n"))
 }
 
 // healthHandler is a simple handler to test if daemon is UP.
 func healthHandler(w http.ResponseWriter, r *http.Request) {
 	w.Header().Add("Content-Type", "application/json")
-	w.Write([]byte(`{"status": "ok"}`))
+	_, _ = w.Write([]byte(`{"status": "ok"}`))
```

Comment:
I think it depends of linter

Reply:
will keep this it does not hurt and make explicitly that we are ignoring the return values